mobius-coroutines/src/test/java/com/spotify/mobius/coroutines/CoroutinesSubtypeEffectHandlerBuilderTest.kt

```diff
@@ -41,7 +38,7 @@ class CoroutinesSubtypeEffectHandlerBuilderTest {
             runTest {
                 val effectHandler = subtypeEffectHandler<Effect, Event>()
 
-                effectHandler.build(coroutineContext)
+                effectHandler.build(UnconfinedTestDispatcher(testScheduler))
```

Comment:
this `UnconfinedTestDispatcher(testScheduler)` is duplicated in multiple tests. Would it be possible to move to a setup method or field initialiser?

Reply:
All tests first create the effect handler, then it is setup and lately it is built using one of the `TestDispatchers`

Since I'm using the functions `advanceUntilIdle()` I need a reference to `testScheduler` from the `TestScope` in the function `runTest`

I tried a couple of options like: 
```kotlin
    // Only build
    private fun CoroutinesSubtypeEffectHandlerBuilder<Effect, Event>.buildUnconfined(testScheduler: TestCoroutineScheduler)
        = this.build(UnconfinedTestDispatcher(testScheduler))
    
    //Build and connect    
    private fun CoroutinesSubtypeEffectHandlerBuilder<Effect, Event>.buildUnconfined(testScheduler: TestCoroutineScheduler, consumer: Consumer<Event> = Consumer {})
        = this.build(UnconfinedTestDispatcher(testScheduler)).connect(consumer)
```

But then the code: 
```kotlin
        effectHandler.build(UnconfinedTestDispatcher(testScheduler))
            .connect { }
            .accept(Effect.Simple)
```
Only gets abbreviated by:
```kotlin
        effectHandler.buildUnconfined(testScheduler))
            .accept(Effect.Simple)
```

without improving readability but obscuring a little bit what is actually happening. 

Do you have any specific function in mind to try ?